lib/components/artwork_grids/generic_grid.js

```diff
@@ -80,7 +80,8 @@ class GenericArtworksGrid extends React.Component {
       const artworkComponents = []
       const artworks = sectionedArtworks[i]
       for (let j = 0; j < artworks.length; j++) {
-        artworkComponents.push(<Artwork artwork={artworks[j]} key={'artwork-' + j} />)
```

Comment:
This is why we really shouldn’t ever be generating keys like this, but use unique identifiers instead.